### PR TITLE
[Deployment] Update Ignite XML Configuration

### PR DESCRIPTION
**Pull request by kevin-zhonghao**
Update Ignite XML Configuration

### CODE REVIEW

File: kubernetes/services/ignite_config.xml [hunk 1]
```diff
@@ -16,45 +15,55 @@
...
   limitations under the License.
 -->
-
 <!--
     Configuration example with Kubernetes IP finder enabled.
 -->
 <beans xmlns="http://www.springframework.org/schema/beans"
-       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xsi:schemaLocation="
+	   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (xieus) [inline kubernetes/services/ignite_config.xml:22]: Try to avoid space.

Comment (kevin-zhonghao) [reply kubernetes/services/ignite_config.xml:22]: Sure

Comment (xieus) [reply kubernetes/services/ignite_config.xml]: @kevin-zhonghao please check the ignite_config.xml. There are more spaces there.

Comment (kevin-zhonghao) [reply kubernetes/services/ignite_config.xml]: Fixed it

**Review by xieus — APPROVED** [review]
LGTM.

**Pull request merged**